tests/testthat/test-compiler-errors.R

```diff
@@ -192,3 +192,10 @@ test_that("animint(out.dir = 'dir1', out.dir = 'dir2') is an error", {
     animint(out.dir = 'dir1', out.dir = 'dir2')
   }, "Duplicate arguments are passed to animint. Duplicate arguments found: out.dir")
 })
+
+test_that("animint() is an error", {
+  
```

Comment:
please remove empty line inside test_that block.
empty lines like this make it more difficult to run the whole block of code interactively. (that can be done with C-c C-c in emacs, etc)